### ISSUE TITLE
`andThen` isn't called if the same mapping has a `then`

### ISSUE THREAD

**Issue by nfcampos** [heroku/react-refetch#110]
Not sure whether we should support this, but we should definitely check that you don't define both on the same mapping, since only `then` will be called.

**Comment by ryanbrainard** [comment]
Hmm, good point. It's a bit unclear what the order would be if we did support it. I agree, let's block it until there is a good reason to support both.

**Comment by nfcampos** [comment]
I need to remember what I was trying to do using both that led me to find it wasn't possible, it's certainly not a common need. In a way you can already do it, I guess you just have to define the `andThen` hook on the `then` mapping, instead of the original

**Comment by ryanbrainard** [comment]
> In a way you can already do it, I guess you just have to define the andThen hook on the then mapping, instead of the original

yeah, if someone really needs it, they can do it explicitly like that.
